dll/win32/kernel32/kernel32.spec

```diff
@@ -93,15 +93,15 @@
 @ stdcall ConsoleMenuControl(long long long)
 @ stdcall ContinueDebugEvent(long long long)
 @ stdcall -stub -version=0x600+ ConvertCalDateTimeToSystemTime(ptr ptr)
-@ stdcall ConvertDefaultLocale(long)
+@ stdcall ConvertDefaultLocale (long)
 @ stdcall ConvertFiberToThread()
 @ stdcall -stub -version=0x600+ ConvertNLSDayOfWeekToWin32DayOfWeek(long)
 @ stdcall -stub -version=0x600+ ConvertSystemTimeToCalDateTime(ptr long ptr)
 @ stdcall ConvertThreadToFiber(ptr)
 @ stdcall ConvertThreadToFiberEx(ptr long)
 @ stdcall CopyFileA(str str long)
-@ stdcall CopyFileExA(str str ptr ptr ptr long)
-@ stdcall CopyFileExW(wstr wstr ptr ptr ptr long)
+@ stdcall CopyFileExA (str str ptr ptr ptr long)
+@ stdcall CopyFileExW (wstr wstr ptr ptr ptr long)
```

Comment:
There are some spaces added, probably by a mistake (and only in kernel32 commit).

Reply:
Fixed